bot.py

```diff
@@ -333,6 +337,32 @@ async def notify_if_voice_client_is_missing(self, message):
             return True
         return False
 
+    async def playlist(self, message, command_content):
+        """Play a playlist"""
+        logging.info("Fetching playlist for user %s", message.author)
+        playlist = pytube.Playlist(command_content)
+        added_videos = 0
+        failed_videos = 0
+        for video in playlist:
+            try:
+                media = pafy.new(video)
+            except KeyError as err:
+                logging.error(err)
+                failed_videos += 1
+                continue
+            self.media_queue.put((media, message))
+            added_videos += 1
+            if added_videos == 1:
+                self.next_in_queue(notify=False)
```

Comment:
Why would we want to skip to the next song here?

Reply:
I'm trying to start the playlist as soon as the first song has been fetched. I should add a check here for whether there is something already playing. Thanks for the catch :)

Reply:
Done.